README.md

```diff
@@ -452,6 +452,20 @@ In your vault go to Settings > Community plugins > Browse and search for "Execut
 or
 
 Follow [this link](https://obsidian.md/plugins?search=execute%20code#) and click "Open in Obsidian".
+	
+## Locating Path Settings ( ex. JavaScript | Node )
+	
+	To avoid or resolve errors from an incorrect path.
+	
+	1. In your terminal, type 'where node'
+	
+	![CleanShot 2022-10-21 at 19 03 49@2x](https://user-images.githubusercontent.com/11916140/197303215-07ff9460-201d-4c0a-95a3-3b1263713a35.png)

```

Comment:
I'm not sure which one is preferred between the markdown `![]()` syntax or `<img>` (ln 468), but I don't think they should be mixed for style reasons. I'm open to any opinions/discussion you have re: this! :)

Reply:
i think that descriptive alt text might help w/ images

Reply:
Yeah -- sorry.   I was just pasting the images in.  I can update that!

Reply:
I don't think it matters, `![]()` is a lot cleaner, using an `<img>` tag is only really necessary for manually setting image dimensions.